bolt/test/match_dwarf.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+
+# This tool helps matching dwarf dumps
+# (= the output from running llvm-objdump --dwarf=frames),
+# by address to function names (which are parsed from a normal objdump).
+# The script is used for checking if .cfi_negate_ra_state CFIs
+# are generated by BOLT the same way they are generated by LLVM.
```

Comment:
Maybe expand with an extra 1-2 lines how this is achieved?

eg, outputting negate offsets – and say why that is a good metric for comparisons.